xla/service/gpu/gpu_windowed_einsum_handler.cc

```diff
@@ -53,11 +53,11 @@ namespace m = match;
 // into
```

Comment:
`Enables the creation of FP8 GEMM Custom Calls for all-gather and
// reduce-scatter windowed einsums in gemm_rewriter.cc...` - Can we add a test that tests the two passes in combination?

Reply:
The comment was intended to indicate that there are dynamic-slice, collective-permute and dot instructions inside the all-gather and reduce-scatter loops. The instructions occur together and don't describe separate cases. I've edited the comment to make it clearer.